server/src/messaging/config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    config: {

```

Comment:
This is the configuration that defines the exchange, queues, bindings and publication. Please check this [rascal](https://github.com/guidesmiths/rascal) doc for more info.